hyperspy/datasets/artificial_data.py

```diff
@@ -356,3 +350,127 @@ def get_atomic_resolution_tem_signal2d():
 
     s = Signal2D(image)
     return s
+
+
+def get_luminescence_signal(navigation_dimension=0,
+                            uniform=False,
+                            add_baseline=False,
+                            add_noise=True,
+                            random_state=None):
+    """Get an artificial luminescence signal in wavelength scale (nm, uniform) or
+        energy scale (eV, non-uniform), simulating luminescence data recorded with a
+        diffracting spectrometer. Some random noise is also added to the spectrum,
+        to simulate experimental noise.
+
+        Parameters
+        ----------
+        navigation_dimension: positive int.
+            The navigation dimension(s) of the signal. 0 = single spectrum,
+            1 = linescan, 2 = spectral map etc...
+        uniform: bool.
+            return uniform (wavelength) or non-uniform (energy) spectrum
+        %s
+        %s
+        random_state: None or int
+            initialise state of the random number generator
+
+        Example
+        -------
+        >>> import hyperspy.datasets.artifical_data as ad
```

Comment:
Typo: artific**i**al_data
```suggestion
        >>> import hyperspy.datasets.artificial_data as ad
```

Reply:
Oopsie, it's corrected